AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -464,8 +470,8 @@ - (void)setAsyncDataSource:(id<ASCollectionDataSource>)asyncDataSource
   if (asyncDataSource == nil) {
     _asyncDataSource = nil;
     _proxyDataSource = _isDeallocating ? nil : [[ASCollectionViewProxy alloc] initWithTarget:nil interceptor:self];
-    
-    memset(&_asyncDataSourceFlags, 0, sizeof(_asyncDataSourceFlags));
+    _asyncDataSourceFlags = {};
+
```

Comment:
I've applied all of your suggestions save this one, with the rationale that:

- This is a higher level of abstraction (symbol vs memory)
- If a dev doesn't understand what `= {}` means then they should look it up and learn, tough love 💪